src/kemal/param_parser.cr

```diff
@@ -60,6 +66,17 @@ class Kemal::ParamParser
     end
   end
 
+  def parse_msgpack
+    body = @request.body as String
+    unpacker = MessagePack::Unpacker.new(body.bytes)
+    case data = unpacker.read
+    when Hash
```

Comment:
yes, I currently not support else (due to a lack of `crystal` / `compiled language` knowledge)
but some othe pr will come and fix this